benefits/static/css/styles.css

```diff
@@ -371,6 +375,10 @@ footer .footer-links li a:visited {
   line-height: 1;
 }
 
+.eligibility-index #login .fallback-text {
+  display: inline-block;
```

Comment:
This is so the `?` icon stays on the same line as the Login.gov image in mobile.

Maybe look at refactoring this rule instead (to not be inside a media query): https://github.com/cal-itp/benefits/blob/dev/benefits/static/css/styles.css#L385